CHANGES.rst

```diff
@@ -140,7 +140,22 @@ Signal
 ------
 - ``metadata.Signal.binned`` is removed, use the ``is_binned`` axis attribute
   instead, e. g. ``s.axes_manager[-1].is_binned``.
+- Some possible values for the ``bins`` argument of the :py:meth:`~.signal.BaseSignal.get_histogram`
```

Comment:
The API documentation seems to be broken in RELEASE_next_major, most method links do not work any more (in contrast to PRs to the RELEASE_next_patch branch).

Reply:
Yes, I have noticed too... anyway, I suspect that the API documentation will need to be improved for the 2.0 as currently, there are a lot of method private method which shouldn't be there! I would suggest to leave as it is for now, and it should be fixed when we revisit the API documentation build.